### ISSUE TITLE
require does not work correctly with typings

### ISSUE THREAD

**Issue by Teascade** [i18next/i18next#1823]
## 💥 Regression Report

When writing in JavaScript, the following code runs correctly with `node test.js`:
```
'use strict';

const i18next = require('i18next');
console.log(i18next.services);
// []
```

However `tsc test.js --allowJs --checkJs --noEmit --esModuleInterop -m commonjs` produces the following error:
```
test.js:5:21 - error TS2339: Property 'services' does not exist on type 'typeof import("/<path>/node_modules/i18next/index")'.

5 console.log(i18next.services);
```

Indicating that the type that typescript received from `require('i18next')` is in fact not `i18n`.

I have not found typescript configurations that would not produce this error.

## Last working version

I don't know, but I've certainly seen [old examples](https://i18next.github.io/i18next/node/index.html) which seem to indicate that this was once working.

## Your Environment

- *runtime version*: node 16 LTS
- *i18next version*: 21.9.1
- *os*: Linux

**Comment by adrai** [comment]
maybe related to https://github.com/i18next/i18next/pull/1352 @rosskevin  ?
// cc @pedrodurek

**Comment by Teascade** [comment]
> maybe related to #1352 @rosskevin ? // cc @pedrodurek

That was merged in version 0.19 and I've now confirmed that this error indeed is produced on version `19.9.2`, but not on `18.0.1`

**Comment by rosskevin** [comment]
What I can say is that I researched extensively before merging #1352, with full research background and repo in #1351.  That was a significant amount of time ago, and I'm not going to dive back into it again.  What I can say is that the route we took with #1352 is now well accepted and the default way of providing typings (and expectations for settings).

So, while I'm not ruling out a bug in the typings, I am stating strongly that a reversal of #1352 should not be under consideration.

@Teascade if it is important to you that your particular approach works, I suggest submitting a PR with breaking tests.  These are run via scripts:

```json
    "test:typescript": "tslint --project tsconfig.json",
    "test:typescript:noninterop": "tslint --project tsconfig.nonEsModuleInterop.json",
```
